src/main/java/com/toofifty/goaltracker/ui/components/ListItemPanel.java

```diff
@@ -104,6 +122,14 @@ public void refreshMenu()
             popupMenu.add(moveToBottom);
         }
         popupMenu.add(removeTask);
+        if (this.indentedListener != null)
+        {
+            popupMenu.add(indentTask);
+        }
+        if (this.unindentedListener != null)
+        {
+            popupMenu.add(unindentTask);
+        }
```

Comment:
Mind moving these into the `!list.isFirst(item)` if statement?

Reply:
Smart!